### DISCUSSION TITLE
Update to the czech locale [cs]

### DISCUSSION THREAD

**Discussion by Tendali** [aclap-dev/video-downloadhelper#894] [Language & Translation]
Hello!

I have translated new added strings regarding reporting errors and leaving a review. Minor spelling mistakes have been also corrected.

Kind regards Tendali
[cs.txt](https://github.com/user-attachments/files/15886531/cs.txt)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
